### ISSUE TITLE
Ugly error messages on failed logins

### ISSUE THREAD

**Issue by kontsevoy** [gravitational/teleport#3774]
### Description

When accessing our staging environments, I tried to use Github or Google SSO and got terrible looking error messages for both cases. (screenshots below

## Problems

Two things are wrong here:

- Error messages are butchered
- Who is "system administrator"? For cloud product that makes no sense.
- Error messages are styled exactly like giant non-clickable buttons. Why so much red? 
- Not helpful. Is this a configuration error, or "access denied" error?

![Screenshot from 2020-05-27 12-47-04](https://user-images.githubusercontent.com/461020/83066178-838e7780-a019-11ea-864d-bd3bee5d2b20.png)
![Screenshot from 2020-05-27 12-44-56](https://user-images.githubusercontent.com/461020/83066190-85f0d180-a019-11ea-865a-e053fd27210e.png)

**Comment by benarent** [comment]
![image](https://user-images.githubusercontent.com/559288/84607724-cba7f980-ae63-11ea-8e0d-b92cf1d436cf.png)

+1 for this. Also running into mobile issues

**Comment by kimlisa** [comment]
@alexwolfe this issue also asks for design and wording change, any input or thoughts?

**Comment by benarent** [comment]
@kimlisa I’ve a plan for a bunch of these. I’ll get them to you tomorrow.

**Comment by benarent** [comment]
I reviewed these, and as we generally provide vague end user error messages to end users. https://github.com/gravitational/teleport/pull/2827 

In discussions with customers, the term `system administrator` isn't ideal and can add extra confusion as who this person is. I chatted with some teams, and some would like the option to provide a direct contact to them as a hyperlink to a Slack Room or Email for support.  We could provide this as an option in the teleport config file, unless there is a better place for setting it.

For now let's.
1. Fix the + issue ( Done )
2. Remove the term `contact your system administrator' from here https://github.com/gravitational/webapps/blob/10993114b632547b300c558743e347a6512c0a9b/packages/design/src/CardError/CardError.jsx#L100 -> Change link to go back to Login page. ( Return to Login ) 
3. Remove system admin from https://github.com/gravitational/teleport/blob/master/lib/auth/oidc.go#L116 https://github.com/gravitational/teleport/blob/master/lib/web/saml.go#L105 
4. Provide input on optional link to customer provided support link.
